### ISSUE TITLE
[FEATURE] @Slf4j Can I add it to the annotation?

### ISSUE THREAD

**Issue by tonycody** [projectlombok/lombok#3457]
![image](https://github.com/projectlombok/lombok/assets/6773121/7a5208f2-5448-4411-9497-b2edc8fc045f)

If yes, the scenarios used are more varied.

**Comment by Rawi01** [comment]
No, that is not possible and not easy to build. This is a duplicate of #557.
